src/include/txn_inline.h

```diff
@@ -1443,13 +1443,13 @@ __wt_txn_id_alloc(WT_SESSION_IMPL *session, bool publish)
     /*
      * Allocating transaction IDs involves several steps.
      *
-     * Firstly, publish that this transaction is allocating its ID, then publish the transaction ID
+     * Firstly, release that this transaction is allocating its ID, then release the transaction ID
      * as the current global ID. Note that this transaction ID might not be unique among threads and
      * hence not valid at this moment. The flag will notify other transactions that are attempting
      * to get their own snapshot for this transaction ID to retry.
      *
      * Then we do an atomic increment to allocate a unique ID. This will give the valid ID to this
-     * transaction that we publish to the global transaction table.
+     * transaction that we release to the global transaction table.
```

Comment:
```suggestion
     * transaction that we release write to the global transaction table.
```

Reply:
This one is actually a bit funky and should perhaps stay as `publish` it is talking about the transaction as a whole.

Reply:
May be change to "set"